### ISSUE TITLE
Add documentation for dexplore library

### ISSUE THREAD

**Issue by Mino260806** [NeonOrbit/ChatHeadEnabler#6]
I have been impressed by the way this module fetches for obfuscated classes and methods dynamically. Can you please consider documenting the little library you implemented to do so ?

**Comment by NeonOrbit** [comment]
Yes, I have plan to make it more powerful, but I was not sure if anyone will be interested. Thanks. 
As for documentation, It's really time consuming and wastes a lot of my time, that's why I tried make my source code as readable as possible. I will try to add important docs for now. Hoping to find a contributor who can help me improve the documentation part. 

Feel free to suggest new features in Dexplore repository, I'll try to implement if possible.
